lib/app_manifest.rb

```diff
@@ -36,9 +36,9 @@ def canonicalize_env(manifest)
       canonicalize_key(manifest, :env) do |env|
         Hash[
           env.map do |key, value|
-            if value.is_a? String
+            if !value.is_a?(Hash)
               value = {
-                value: value,
+                value: value.to_s,
```

Comment:
This coercion is different from what Nyata currently expects, [see here](https://github.com/heroku/nyata/blob/master/lib/nyata/manifest/validator.rb#L64-L76). My worry here is we will coerce a value like the following into a string:

```
"env": {
  "HELLO": [1, 2, 3, "lol"]
}
```